source/guides/recipes/dependency-management.md

```diff
@@ -80,7 +80,7 @@ This will result in a different version being built:
 ```shell-session
 $ nix-build -A build
 $ ./result/bin/hello --version | head -1
-hello (GNU Hello) 2.10
+hello (GNU Hello) 2.12
```

Comment:
Tested this, but 23.05 has the same version as 23.11, `2.12.1`, so the above text in this tutorial doesn't work:

> This will result in a different version being built

```suggestion
hello (GNU Hello) 2.10
```